src/hoi4_world/countries/hoi4_country.h

```diff
@@ -69,6 +73,8 @@ class Country
    std::optional<int> capital_state_;
    std::string ideology_ = "neutrality";
    std::string sub_ideology_ = "despotism";
+   std::map<std::string, int> ideology_support_{std::make_pair("neutrality", 100)};
```

Comment:
You might be able to do
std::map<std::string, int> ideology_support_{{"neutrality", 100}};